pkg/provisioner/ironic/ironic.go

```diff
@@ -802,6 +807,19 @@ func (p *ironicProvisioner) setLiveIsoUpdateOptsForNode(ironicNode *nodes.Node,
 		)
 	}
 
+	// FIXME(shardy) hardcoded for testing, add an env var
+	forcePersistentBootDevice := "Never"
+	if ironicNode.DriverInfo["force_persistent_boot_device"] != forcePersistentBootDevice {
+		updates = append(
+			updates,
+			nodes.UpdateOperation{
+				Op:    nodes.AddOp,
+				Path:  "/driver_info/force_persistent_boot_device",
+				Value: forcePersistentBootDevice,
+			},
+		)
+	}
```

Comment:
Note that on the master branch we'll want to use updater.SetTopLevelOpt() for everything.

Reply:
Thanks, I added a function to merge driver_info instead ref https://github.com/metal3-io/baremetal-operator/pull/949